packages/mermaid/src/diagrams/flowchart/flowDb.js

```diff
@@ -192,6 +192,12 @@ export const updateLinkInterpolate = function (positions, interp) {
  */
 export const updateLink = function (positions, style) {
   positions.forEach(function (pos) {
+    if (pos >= edges.length) {
+      let error = new Error(
+        `The index for linkStyle is out of bounds. (Help: Ensure that the index is within the range of existing edges.)`
+      );
+      throw error;
```

Comment:
```suggestion
      throw new Error(
        `The index for linkStyle is out of bounds. (Help: Ensure that the index is within the range of existing edges.)`
      );
```

Reply:
You could also add the values in the error message. 

`The index for linkStyle is out of bounds. Index ${pos} is greater than the available edges ${edges.length}.  (Help: Ensure that the index is within the range of existing edges.)`

Reply:
Hi @sidharthv96 , Thanks for your valuable suggestions, 
I am using  

**The index ${pos} for linkStyle is out of bounds. Valid indices for linkStyle are between 0 and ${edges.length - 1}. (Help: Ensure that the index is within the range of existing edges.)**,

 this as an error messgae , because _above error message is not very intuitive I believe in case of 
 
 **The index for linkStyle is out of bounds. Index 1 is greater than the available edges 1.  (Help: Ensure that the index is within the range of existing edges.)**

But still if you want to me to use the above one please let me know